### PR TITLE
Feature/navbar component

### PR DESCRIPTION
**Pull request by DrewBradley**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Builds out the NavBar component to contain links to the Calendar, Form, and Contacts pages.
Uses 'Font Awesome' icons to provide different icon options.

## *Instructions for reviewer:*
*Where should the reviewer start?*
`src/components/App.js`
`src/components/navbar/NavBar.css`
`src/components/navbar/NavBar.js`

*How should this be manually tested?*
run `npm install` to ensure you have the necessary dependencies.
Run `npm start` and open the project on `localhost:3000`.

*Any background context you want to provide?*
Font Awesome provides a variety of SVG icons so the icon size can be adjusted or rendered differently depending on the device.
Screen Shot 2021-04-08 at 5 58 37 PM

*What are the relevant tickets?*
#7 

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
Is Font Awesome too much for a hassle to use?
https://fontawesome.com/how-to-use/on-the-web/using-with/react

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

**Review by kristenmb — APPROVED** [review]
This all looks great. I am not in love with the icons style-wise. But functionality wise this is perfect.

**Review by kevinhartmann23 — APPROVED** [review]
Agreed with @kristenmb ! I think we can tweak the icons, but I perfect start and functionality. Thank you @DrewBradley !

File: src/components/App.js [hunk 1]
```diff
@@ -1,17 +1,20 @@
 import React from 'react'
 import {Switch, Route} from 'react-router-dom'
 import './App.css';
-import Dashboard from './Dashboard'
-import Header from './Header'
+// import Dashboard from './Dashboard'
```
Comment (richardltyler) [inline src/components/App.js:4]: I appreciate the commented out setup

**Review by richardltyler — APPROVED** [review]
Looks awesome!

**Pull request merged**